src/Uno.Foundation/Size.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using System.ComponentModel;
+using System.Diagnostics;
 using System.Security;
 
 namespace Windows.Foundation
 {
+	[DebuggerDisplay("{Width}x{Height}")]
```

Comment:
Same comment, `ToString` already does that, and at some point I would prefer if we have the same output for the `ToString` (logs) and the debugger display

Reply:
It's not working everywhere in VS. The DebuggerDisplay, yes.